x-pack/plugins/monitoring/server/alerts/base_rule.ts

```diff
@@ -344,6 +344,9 @@ export class BaseRule {
     if (ccs) {
       globalState.push(`ccs:${ccs}`);
     }
-    return `${Globals.app.url}/app/monitoring#/${link}?_g=(${globalState.toString()})`;
+
+    const baseUrl = Globals.app.url || '(server.publicBaseUrl)';
+
+    return `${baseUrl}/app/monitoring#/${link}?_g=(${globalState.toString()})`;
```

Comment:
Since the input may be coming from the user (cloud sets the kibanaBaseUrl and it's not overridable by customers), would be best to use something like a relative URL "joiner" (not sure what the current flavor would be).  Reason is, the customer may or may not be adding an trailing `/` to the URL, and you don't want to generate `...//app/...` (though that might well work in some envs) 

I believe I've cheated and used `path.join()` for this as well, but ... using a `url` or `URL` or whatever is correct these days would be better.

Reply:
Ah, if that's the case, shouldn't need a special join!